### ISSUE TITLE
Setup script exited with error in anyjson setup command: use_2to3 is invalid.

### ISSUE THREAD

**Issue by georgerappel** [LibreTime/libretime#1328]
**Describe the bug**
When doing a clean install on a cloud provider (tried both AWS and Azure), using the current master (509f680e6d9b7166a409fe5127d60def5023edbe), the Airtime configuration fails on a python dependency installation with the following error:

```
Installed /usr/local/lib/python3.6/dist-packages/airtime_playout-1.0-py3.6.egg
Processing dependencies for airtime-playout==1.0
Searching for anyjson
Reading https://pypi.org/simple/anyjson/
Downloading https://files.pythonhosted.org/packages/c3/4d/d4089e1a3dd25b46bebdb55a992b0797cff657b4477bc32ce28038fdecbc/anyjson-0.3.3.tar.gz#sha256=37812d863c9ad3e35c0734c42e0bf0320ce8c3bed82cd20ad54cb34d158157ba
Best match: anyjson 0.3.3
Processing anyjson-0.3.3.tar.gz
Writing /tmp/easy_install-1v9sfm3h/anyjson-0.3.3/setup.cfg
Running anyjson-0.3.3/setup.py -q bdist_egg --dist-dir /tmp/easy_install-1v9sfm3h/anyjson-0.3.3/egg-dist-tmp-7dgdyjbj
error: Setup script exited with error in anyjson setup command: use_2to3 is invalid.
```

**To Reproduce**

1. Launch a new virtual machine on Azure or EC2 on AWS with Ubuntu 18.04 OR Debian 10.
2. Install LibreTime following the guide on the website - https://libretime.org/install

**Expected behavior**
Installation success

**Installation method:**

- OS: Ubuntu
- OS Version: both on 18.04.5 LTS and on Debian 10.
- Method: `./install` script
- Details: Installed through the command line following the tutorial. I have also tried installing locally with pip/python and system libraries, with no success, as it seems that for the virtual env it downloads the repositories again.
- I have also read that Azure does not contain the fix_ file sources for the `2to3` python library. Testing for some hours with the few solutions I found online, I couldn't get anywhere.
- Given that LibreTime is for radios, running it on a server seems essential, thus this is a big entry barrier.

**Comment by jooola** [comment]
I can reproduce this on Ubuntu 18.04, I 'll take a look.

**Comment by paddatrapper** [comment]
Pypi... Always fun when your dependencies change out from under you

**Comment by jooola** [comment]
The `anyjson` package on pypi is an old package which hasn't completely been ported to python3. Ubuntu bionic ships a updated version with full python3 compatibility.

The question is whether we introduced this bug or it comes from upstream.

**Comment by jooola** [comment]
A fix is to install the `python3-anyjson` package by hand and rerun the installation script.

**Comment by jooola** [comment]
I'll try to graph the dependencies we removed from the deps manager change. Maybe python3-anyjson is in the middle of this.

**Comment by jooola** [comment]
Found the culprit, and it is indeed a pypi package update that broke our install script. But I think this is actually a good thing.

So the latest version of setuptools is dropping support of the `use_2to3` flag. See https://github.com/pypa/setuptools/blob/main/CHANGES.rst#v5802

Since we install a fresh version of setuptools from pip, we ended up with this issue.

https://github.com/LibreTime/libretime/blob/2513f08d0774a5430254c10e2b6eb2b870719ba7/install#L1001

@paddatrapper Should we go for installing the `python3-anyjson` distro package, and maybe consider #1154 already ?
